src/item.cpp

```diff
@@ -4341,68 +4341,91 @@ item_location item::pick_reload_ammo( player &u, bool interactive ) const
     }
 
     // If interactive and more than one option prompt the user for a selection
-    uimenu amenu;
-    amenu.return_invalid = true;
-    amenu.w_y = 0;
-    amenu.w_x = 0;
-    amenu.w_width = TERMX;
-    // 40: item location
-    // 40: = 4 * ammo stats colum (10 chars each)
-    // 2: prefix from uimenu: hotkey + space in front of name
-    // 4: borders: 2 char each ("| " and " |")
-    const int namelen = TERMX - 40 - 40 - 2 - 4;
+    uimenu menu;
+    menu.text = string_format( _("Reload %s" ), tname().c_str() );
+    menu.return_invalid = true;
+
+    // Construct item names
+    std::vector<std::string> names;
+    std::transform( ammo_list.begin(), ammo_list.end(), std::back_inserter( names ), []( item_location& e ) {
+        const item *it = e.get_item();
+        if( it->is_magazine() && it->ammo_data() ) {
+            return string_format( "%s with %s (%d)", it->type->nname( 1 ).c_str(),
```

Comment:
Agreed, this needs to be localizable, and ideally should also have a translator comment (because that is a lot of formatting codes), along the lines of
```C++
//~ "[magazine] with [ammo] ([##])"
return string_format( _("%s with %s (%d)"), it->type->nname( 1 ).c_str(),
```